integrationTests/state/stateTrie_test.go

```diff
@@ -1552,6 +1553,7 @@ func testNodeStateCheckpointSnapshotAndPruning(
 
 	assert.Equal(t, 5, len(prunedRootHashes))
 	for i := range prunedRootHashes {
+		fmt.Println(fmt.Sprintf("Should be pruned!!! hash: %v", core.ToHex(prunedRootHashes[i])))
```

Comment:
To many ! Might consider using logger infrastructure

Reply:
removed